camptix.php

```diff
@@ -3191,7 +3191,10 @@ function menu_tools_notify() {
 						<# }); #>
 					</select>
 					<# } else if ( data.type == 'text' ) { #>
-					<input type="text" class="segment-value regular-text" value="{{ data.model.value }}" />
+					<input placeholder="{{ data.placeholder }}" type="text" class="segment-value regular-text" value="{{ data.model.value }}" />
+						<# if ( data.placeholder.length > 0 ) { #>
+							<span class="screen-reader-text">{{ data.placeholder }}</span>
+						<# } #>
```

Comment:
Since this screen reader text element isn't actually linked to the input, it may not be that useful. For now, I'd say let's remove it, and in a separate PR we could try implementing proper `<label>` elements for this form, and perhaps other accessibility improvements.

Reply:
Thanks for the review @coreymckrill. I've amended the commit that reflects the changes you requested. 

https://github.com/Automattic/camptix/pull/242/files#diff-5cb4f0974d7e507f6aa5827e23e12aadL3191-L3197